malaria24/ona/views.py

```diff
@@ -24,6 +25,14 @@ def localities(request, facility_code):
         raise Http404()
 
 
+def health(request):
+    status = 200
+    resp = {
+        "up": True,
+    }
+    return HttpResponse(resp, status=status)
```

Comment:
pls use JsonResponse instead, so that it stays consistent with what the old /health/ endpoint did
https://docs.djangoproject.com/en/dev/ref/request-response/#id5
(Since it subclasses HttpResponse you can probably still send the `status` argument)